src/entrypoints/content.ts

```
@@ -231,15 +232,18 @@ export default defineContentScript({
+          const defuddle = new Defuddle(window.document);
+          const dedocument = defuddle.parse();
           const context: AgentContext = {
             sourceText,
             targetLanguage: 'zh-CN',
-            siteTitle: document.title,
-            siteUrl: document.location.href,
+            siteTitle: dedocument.title,
+            siteUrl: window.location.href,
+            siteDescription: dedocument.description,
           };
           logger.debug`context ${{ context }}`;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)